engines/python/setup/djl_python/tests/test_rolling_batch.py

```diff
@@ -37,6 +37,53 @@ def test_json_fmt(self):
             self.assertEqual(json.dumps({"generated_text": "Hello world"}),
                              req.get_next_token())
 
+
+def test_last_token_is_first_token(self):
+    req_input1 = TextInput(
+        request_id=0,
+        input_text="This is a wonderful day",
+        parameters={
+            "max_new_tokens": 256,
+            "details": True
+        },
+        output_formatter=_json_output_formatter,
+    )
+    req1 = Request(req_input1)
+    req_input2 = TextInput(
```

Comment:
What's the diff between req_input1 and req_input2? Both are using the same output_formatter.

Reply:
There looks like an indentation problem.

Reply:
ah sorry, i think i was playing around with Stream and forgot to remove the test. there's no reason to have req 2 if stream isn't making any functional difference.